geo-traits/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "geo-traits"
+version = "0.1.0"
+license = "MIT OR Apache-2.0"
+repository = "https://github.com/georust/geo"
+documentation = "https://docs.rs/geo-traits/"
+readme = "../README.md"
+keywords = ["gis", "geo", "geography", "geospatial"]
+description = "Geospatial traits"
+rust-version = "1.65"
+edition = "2021"
+
+[features]
+
+[dependencies]
+geo-types = "0.7.9"
```

Comment:
What do you think about making `geo-types` optional? In theory if someone wanted to just implement `geo-traits` for their own geometry types, they shouldn't need access to `geo-types`, I think

Reply:
Hmm actually this depends on `CoordTrait`. Maybe that should live in this crate someday 🤔

Reply:
Yeah I think making `geo-types` optional definitely makes sense but in this initial implementation I made a hard dependency for simplicity